### PR TITLE
Added test automation for BZ 1664435

### PR DESCRIPTION
**Pull request by gswami90**
Test Case Added for BZ 1664435

1] Update Existing ca.cfg with below parameters.

ca.scep.enable=true
ca.scep.allowedEncryptionAlgorithms=DES
ca.scep.allowedHashAlgorithms=MD5
ca.scep.encryptionAlgorithm=DES
ca.scep.hashAlgorithm=MD5

2] SCEP Certificate Enroll,

- Get CA certificate
sscep getca -c ca.crt -u 'http://<host>:<port>/ca/cgi-bin/pkiclient.exe'

- Generate keys and cert request
mkrequest -ip <IP Address> redhat

- Submit request
sscep enroll -u http://<host>:<port>/ca/cgi-bin/pkiclient.exe -c ca.crt -k local.key -r local.csr -l cert.crt

Signed-off-by: Gaurav Swami <gswami@redhat.com>

### CODE REVIEW

File: tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+	# Validate the enrolled cert for fingerprint match
+	cmd = ansible_module.shell('openssl x509 -in cert.crt -text -noout')
+	for result in cmd.values():
+		assert 'Signature Algorithm: sha512WithRSAEncryption' in result['stdout']
+		log.info('Successfully matched the sha512 fingerprint')
+	time.sleep(5)
+	
+	# Remove the generated cert and key
+	ansible_module.shell('rm -rf local.key local.csr cert.crt ca.crt')
+	log.info('Successfully removed the cert and key from server')
```
Comment (06shalini) [inline tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py:182]: New line missing

File: .github/workflows/qe-tests.yml [hunk 2]
```diff
@@ -109,6 +109,15 @@ jobs:
             /vagrant/tests/dogtag/pytest-ansible/pytest/sanity/test_role_users.py \
             --junit-xml pki_cert_junit.xml -qsvvv'
 
+        - name: Run SCEP tests
```
Comment (06shalini) [inline .github/workflows/qe-tests.yml:112]: Remove this section before merge

**Review by 06shalini — COMMENTED** [review]
Looks good to me, if it passes pipeline.

File: tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py (hunk 1 above)
Comment (gswami90) [reply tests/dogtag/pytest-ansible/pytest/ca/scep/test_bug_1664435_pki_ca_scep_enrollment.py]: Added new line.

**Review by 06shalini — APPROVED** [review]
Looks good to me. 
Please revert qe-tests.yml  changes and merge.
Requirement also need update.

**Comment by gswami90** [comment]
> Looks good to me.
> Please revert qe-tests.yml changes and merge.
> Requirement also need update.

I have reverted the changes from qe-tests.yml file.

**Pull request merged**